library/Templates/General/hello.html

```diff
@@ -0,0 +1,8 @@
+<!DOCTYPE html>
+<html lang="EN">
```

Comment:
I suggest to include also "subject" for HTML templates.

Reply:
Not yet resolved.

Reply:
@ateska Let me know if you are happy with the HTML template

Reply:
Do you have example somewhere with the Subject being part of the HTML template?

Reply:
@ateska `def find_subject_in_html(body):
	regex = r"(<title>(.*)</title>)"
	match = re.search(regex, body)
	if match is None:
		return body, None
	subject_line, subject = match.groups()
	body = body.replace(subject_line, "")
	return body, subject
`

I think title is assumed to be subject , correct?

https://github.com/TeskaLabs/asab-iris/blob/5e32bffe0093386fc10bde6b3843eec2740db580/library/Templates/Email/hello.html#L3

Reply:
Yes, remove `-` and it is correct.